crates/next-core/js/src/overlay/internal/container/Errors.tsx

```diff
@@ -177,8 +198,15 @@ export function Errors({ issues, errors }: ErrorsProps) {
     }
   }, [defaultTab]);
 
+  const onlyHasWarnings = !hasErrors && !hasIssueWithError;
+
+  // eslint-disable-next-line prefer-const
+  let [displayState, { fullscreen, minimize, hide }] = useDisplayState(
+    onlyHasWarnings ? DisplayState.Minimized : DisplayState.Fullscreen
+  );
```

Comment:
You could always do:
```suggestion
  const [stateDisplayState, { fullscreen, minimize, hide }] = useDisplayState(
    onlyHasWarnings ? DisplayState.Minimized : DisplayState.Fullscreen
  );
  let displayState = stateDisplayState;
```

Reply:
hmm, not super great, but I'll do that for now